README.md

```diff
@@ -40,6 +40,11 @@ Additionally, a couple _amazing_ people have started to work on an
 * [Setup by adi1090x](https://github.com/adi1090x/widgets)
 ![Nordic](https://raw.githubusercontent.com/adi1090x/widgets/main/previews/nordic.png)
 
+* [i3 Bar replacement by owenrumney](https://github.com/owenrumney/eww-bar)
+
```

Comment:
this blank line here breaks the markdown indentation ^^ make sure the images are directly after the corresponding bulletpoint

Reply:
sure think, that should be fixed now